starter-code/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="main.css">
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js" integrity="sha384-a5N7Y/aK3qNeh15eJKGWxsqtnX/wWdSZSKp+81YjTmS15nvnvxKHuzaWwXHDli+4" crossorigin="anonymous"></script>
+
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy" crossorigin="anonymous">
+
+    
+    <title>Revera</title>
+</head>
+<body>
+    <nav class="navbar">
+        <h1>Revera</h1>
+        <ul>
+            <li>Home</li>
+            <li>About</li>
+            <li>Sample page</li>
+            <li>Portfolio</li>
+            <li>Blog</li>
+                    <li role="presentation" class="dropdown">
+                      <a class="dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">
+                        Parent page <span class="caret"></span>
+                      </a>
+                      <ul class="dropdown-menu">
+                        <div><li>Full Page</li></div>
+                        <div><li>Child Page 1</li></div>
+                        <div><li>Child Page 2</li></div>
+                      </ul>
+                    <li>Contact</li>
+        </ul>
+    </nav>
+
+        <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+            <div class="carousel-inner">
+                <div class="carousel-item active">
+                    <img class="d-block w-100" src="img/slider-1.jpg" alt="First slide">
+                </div>
+                <div class="overlay"></div>
+                <div class="carousel-item">
+                    <img class="d-block w-100" src="img/slider-2.jpg" alt="Second slide">
+                </div>
+                <div class="carousel-item">
+                     <img class="d-block w-100" src="img/slider-3.jpg" alt="Third slide">
+                </div>
+                <div class="carousel-item">
+                     <img class="d-block w-100" src="img/slider-4.jpg" alt="Third slide">
+                </div>
+            </div>
+            <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
```

Comment:
Arrows are not working for me. Bootstrap is also using jQuery, so you may have something broken related to this jquery.